jenkins/run-server-func-tests

```diff
@@ -72,6 +73,14 @@ stop_status=${?}
 if [[ ${exit_status} -eq 0 ]]; then
     exit_status=${stop_status}
 fi
+
+# Remove the server container
+podman stop ${PB_SERVER_CONTAINER_NAME}
+stop_status=${?}
+if [[ ${exit_status} -eq 0 ]]; then
+    exit_status=${stop_status}
+fi
```

Comment:
This looks good, but consider the sequencing:  we're stopping the pod _before_ stopping the Pbench Server, which could be awkward if the Server tries to make a request of one of the services that we just stopped.

We really should shut things down in (reverse-)dependency order -- that is, in the opposite of the order that we started them up.  So, we **should** shut down the Pbench Server first, and _then_ the services that it depends on.

Reply:
Thats a valid point and I am tempted to change it even though I have an approval :stuck_out_tongue_closed_eyes:

Reply:
Fixed.